teamA/jeongDoYoung/routes/index.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+const postRouter = require("./postRouter");
+router.use("/users", userRouter.router);
+router.use("/posts", postRouter.router);
```

Comment:
외부 모듈을 불러오는 코드와 모듈의 속성을 지정하는 코드 사이에 공백을 넣어 가독성을 높여주세요!
```suggestion
const userRouter = require("./userRouter");
const postRouter = require("./postRouter");

router.use("/users", userRouter.router);
router.use("/posts", postRouter.router);
```